src/module/chat-message/data.ts

```diff
@@ -22,12 +22,19 @@ export interface ItemOriginFlag {
     rollOptions?: string[];
 }
 
+export interface ChatCastingFlag {
```

Comment:
export at bottom of file

Reply:
the interface just above was using `export` directly so i just mimicked, i will do it for both then